uploadNeuroDB/imaging_upload_file.pl

```diff
@@ -279,6 +286,11 @@ =head2 Methods
     $message = "\nThe candidate info validation has failed.\n";
     spool($message,'Y', $notify_notsummary);
     print STDERR $message;
+    open MAIL, "| mail $mail_user";
+    print MAIL "Subject: IMAGING_UPLOAD_FILE: ".$uploaded_file." insertion completed.\n\n";
```

Comment:
Optional/improvement - Making this a function that'll take parameters (i.e. $mail_user, $subject, $message) , instead of repeating it.

Reply:
would this function be defined in this same file?

Reply:
@mohadesz @nicolasbrossard I am thinking that maybe it could go in Notify.pm? Would it make sense?

I know Notify.pm is used to write into the notification_spool table so maybe this is not the right place... 

Or ImagingUpload.pm? But then, this function could only be access via the imagingUpload object...

The other library I could think of is MRI.pm but that seems a little random I guess.

I don't know, lol.

Reply:
@cmadjar  if it's added to notify.pm can it be accessed from anywhere? If so, maybe notify.pm makes more sense

Reply:
once added we could do a quick clean up - to make sure every instance where "open Mail" is used will be replaced by the function with the appropriate parameters. This is more like a wishlist lol

Reply:
@mohadesz @cmadjar Yeah, putting this in Notify.pm makes sense.